src/modules.py

```diff
@@ -163,7 +163,7 @@ def parse_hawkeye(config: Dict[str, Any]) -> IocCollection:
 @module("trickbot")
 def parse_trickbot(config: Dict[str, Any]) -> IocCollection:
     iocs = IocCollection()
-    if config["public_key"]:
+    if "public_key" in config:
```

Comment:
```suggestion
    if config.get("public_key"):
```

Reply:
Should we expect false-y values inside the key?

Reply:
No, I've used the pattern above for consistency with other modules - but both (should be) fine I guess